src/components/Converters/ClassConverter.ts

```diff
@@ -67,16 +67,20 @@ export class ClassConverter extends RecordConverter {
         const interfaceRows: string[] = [];
         const TAB = SpecialCharacterHelper.TAB;
 
-        interfaceRows.push(`export interface ${data.name}${this.interfaceSuffix} {`);
-        rows.push(`export class ${data.name} extends BaseAvroRecord implements ${data.name}${this.interfaceSuffix} {`);
+        let name = data.name;
```

Comment:
do you think it makes sense to call this variable `fullname` to better disambiguate if it is with or without `namespace`?

Reply:
yes, and i'll do the same for fieldName which gets passed to `getField()`